File: test_server.py
from flask import Flask, jsonify
from waitress import serve

app = Flask(__name__)


@app.route("/")
def index():
    return jsonify({"status": "ok", "message": "Test server is running"})


if __name__ == "__main__":
    print("Starting test server on http://localhost:8080")
    serve(app, host="localhost", port=8080)

File: test-server.js
const express = require('express');
const cors = require('cors');

const app = express();
const port = 8080;

app.use(cors());
app.use(express.json());

app.get('/', (req, res) => {
  res.send('DojoPool Test Server - Working!');
});

app.get('/api/health', (req, res) => {
  res.json({
    status: 'ok',
    timestamp: new Date().toISOString(),
    message: 'Backend server is running',
  });
});

app.get('/api/test', (req, res) => {
  res.json({ message: 'Test endpoint working!' });
});

app.listen(port, () => {
  console.log(`🚀 Test server running on http://localhost:${port}`);
  console.log(`📊 Health check: http://localhost:${port}/api/health`);
  console.log(`🧪 Test endpoint: http://localhost:${port}/api/test`);
});
